### PR TITLE
Test Boiler

### PR DESCRIPTION
**Pull request by cristianfchq**
add test for class Boiler and added some methods in Boiler.java file

### CODE REVIEW

**Review by danielaSantaCruz — APPROVED** [review]
It's great

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java [hunk 1]
```diff
@@ -1,12 +1,89 @@
...
+    public void getCupsOfWater_Zero() {
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(0);
+        int expected = 0;
+        int actual = boiler.getCupsOfWater();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void getCapacity_Twelve() {
```
Comment (edsonarios) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: the name of the method is glasses of water, but the test indicates capacity, it is inconsistent

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java [hunk 2]
```diff
@@ -1,12 +1,89 @@
...
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(12);
+        int expected = 12;
+        int actual = boiler.getCupsOfWater();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setCapacity_NewCapacityZero_Zero() {
```
Comment (edsonarios) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: the name of the method is glasses of water, but the test indicates capacity, it is inconsistent

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java [hunk 3]
```diff
@@ -1,12 +1,89 @@
...
+        Boiler boiler = new Boiler();
+        boiler.setCupsOfWater(0);
+        int expected = 0;
+        int actual = boiler.getCupsOfWater();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void setCapacity_NewCapacityTwelve_Twelve() {
```
Comment (edsonarios) [inline src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: the name of the method is glasses of water, but the test indicates capacity, it is inconsistent

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java (hunk 1 above)
Comment (cristianfchq) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: that's correct. I changed it edson

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java (hunk 2 above)
Comment (cristianfchq) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: that's correct. I changed it edson

File: src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java (hunk 3 above)
Comment (cristianfchq) [reply src/test/java/org/fundacionjala/prog101CoffeeMaker/coffeeMaker/testBoiler/BoilerTest.java]: that's correct. I changed it edson

**Review by edsonarios — APPROVED** [review]

**Pull request merged**